app/components/Notification/index.jsx

```diff
@@ -54,7 +56,7 @@ let NotifiItem = React.createClass({
                         <div className="name" style={nameStyle}><Name member={this.props.sender}/></div>
                         <div className="time" style={timeStyle} flex={1}><SmartTimeDisplay start={this.props.time} relative/></div>
                     </Layout>
-                    <div style={messageStyle}>{this.props.message}</div>
+                    <a href={this.props.url} style={messageStyle}>{this.props.message}</a>
```

Comment:
`<a>` will cause the whole page refresh, which is not good UX.

Consider to use react router and do transition without refreshing the whole page.